client/image.go

```diff
@@ -35,3 +38,28 @@ func (client *APIClient) ImageList() ([]types.Image, error) {
 	return imageList, err
 
 }
+
+// ImageRemoves deletes images.
+func (client *APIClient) ImageRemoves(name string, force bool) error {
+	q := url.Values{}
+	if force {
+		q.Set("force", "true")
+	}
+
+	resp, err := client.delete("/images/"+name, q)
+	if err != nil {
+		return err
+	}
+
+	defer ensureCloseReader(resp)
+
+	if resp.StatusCode != http.StatusNoContent {
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return err
+		}
+		return errors.New(string(body))
```

Comment:
if string(body) is empty string, this function will return an error. Is it reasonable.
Actually if  `resp.StatusCode != http.StatusNoContent` returns true, it means this is a non-2xx response. Then the error message will not be in `resp`, I think it is in `err` of `resp, err := client.delete("/images/"+name, q)`. This is guaranteed by HTTP framework. So I am wondering if this block is reasonable. 
Also we found similar code in other function of api client. Yes, if you agree with me, we should reconsider that part as well. @HusterWan @skoo87

Reply:
@allencloud you are right, maybe we can return err directly here
``` 
_, err := client.delete("/images/"+name, q)

return err
```